toolbox/certbot/renew.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 
+echo ""
+echo "$(date '+%Y-%m-%d %H:%M:%S')"
 echo "Renewing the certificates for the Airflow stack"
+
+
 /usr/airflow/atd-airflow/toolbox/certbot/renew_domain_with_certbot.sh airflow.austinmobility.io
 /usr/airflow/atd-airflow/toolbox/certbot/renew_domain_with_certbot.sh airflow-workers.austinmobility.io
 
-# Build the HAProxy image with the renewed certificates
 cd /usr/airflow/atd-airflow
-docker compose build haproxy
```

Comment:
We use the `haproxy` image directly; we're no longer building our SSL certificates into the image, so we don't need to build anything here.